content/cpp/concepts/math-functions/terms/modf/modf.md

```diff
@@ -0,0 +1,77 @@
+---
+Title: 'modf()'
+Description: 'Splits the given number into an integral and a fractional part.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`modf()`** function splits the given number into an integral and a fractional part. The function stores the integral part in the object passed to `modf()` as an argument. The function returns the fractional part.
+
+## Syntax
+
+```pseudo
+modf(number, integralPart)
+```
+
+The `modf()` function takes two parameters:
+
+- `number`, a floating point value (`double`, `float` or `long double`) to split into integral and fractional parts.
+- `integralPart`, a pointer to floating point value to store the integral part. It will have the same type and sign as the `number`.
+
+The `modf()` function returns:
```

Comment:
This line and line 29 can be combined into one sentence like so:

"The `modf()` function returns the fraction part of `number` with the same sign as `number`."